python/pytests/execution_test.py

```diff
@@ -34,14 +34,27 @@ async def test_iter_pandas(golden, source_int64) -> None:
         next(batches)
 
 
-async def test_iter_rows(golden, source_int64) -> None:
+async def test_iter_rows(source_int64) -> None:
     results: Iterator[dict] = source_int64.run_iter("row", row_limit=2)
     assert next(results)["m"] == 5
     assert next(results)["m"] == 24
     with pytest.raises(StopIteration):
         next(results)
 
 
+@kd.udf("add_one<N: number>(x: N) -> N")
+def add_one(x: pd.Series) -> pd.Series:
+    """Use Pandas to one"""
+    print("In UDF")
+    return x + 1
+
+
+async def test_iter_udf(source_int64) -> None:
+    results = source_int64.col("m").pipe(add_one).run_iter("row", row_limit=2)
+    assert next(results)["result"] == 6
+    assert next(results)["result"] == 25
+
+
```

Comment:
I would have expected a non-async test.

Reply:
ah, maybe the test is async but the call to `run_iter` is non-async

Reply:
I think all of our tests require `async` because under the hood some of the APIs use async.